services/device-registry-mongodb/src/main/java/org/eclipse/hono/deviceregistry/mongodb/config/MongoDbConfigProperties.java

```diff
@@ -182,7 +183,7 @@ public MongoDbConfigProperties setConnectionString(final String connectionString
      * Gets the time in milliseconds that the mongo driver will wait to select a
      * server for an operation before raising an error.
      * <p>
-     * When this property is not set, the Vert.x mongodb client uses a default value of 30000 ms.
+     * When this property is not set, the Vert.x mongodb client uses a default value of 1000 ms.
```

Comment:
you can use the `{@value }` tag in order to refer to a constant's value. That way you do not need to explicitly spell it out ...

Reply:
@sophokles73 Sounds good. The corresponding changes have been made.